mempool/tx_pool_test.go

```diff
@@ -173,7 +174,7 @@ func createBatch(t testing.TB, signer *signing.EdSigner) ([]*types.Transaction,
 	var txBatch []*types.Transaction
 	var txIDBatch []types.TransactionID
 	for i := uint64(0); i < 10000; i++ {
-		tx, err := types.GenerateCallTransaction(signer, types.Address{}, 5+1, 50, 100, 1)
+		tx, err := transaction.GenerateCallTransaction(signer, types.Address{}, 5+1, 50, 100, 1)
```

Comment:
No spawn transactions in this file?

Reply:
No, because they are not submitted. There's no nonce == 0 in this file, so as I understand, there's no need for spawn TXs here. Please correct me if I'm wrong